probes/entries.go

```diff
@@ -12,11 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//nolint // to prevent linter errors because of package names being camel case
```

Comment:
Nit: determine the particular `golangci-lint` check that fails, and rewrite this nolint as `nolint:<checkname>`.

Reply:
fixed